traitsui/qt4/tabular_editor.py

```diff
@@ -517,19 +517,31 @@ def _selected_column_changed(self, selected_column):
     def _scroll_to_row_changed(self, row):
```

Comment:
if we convert `scroll_to_row_hint` into a property, we will want to replace all uses of `scroll_to_row_hint` internally with `scroll_to_position_hint` instead.